<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GraphicsSettingDialog</class>
 <widget class="QDialog" name="GraphicsSettingDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>248</width>
    <height>375</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Graphics Setting</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <item>
    <widget class="QGroupBox" name="elevationPointsGroupBox">
     <property name="title">
      <string>Elevation Points</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QGroupBox" name="elevationPointSizeGroupBox">
        <property name="title">
         <string>Size</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QRadioButton" name="elevationPointsAutoSizeRadioButton">
           <property name="text">
            <string>Auto</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="elevationPointsManualSizeRadioButton">
           <property name="text">
            <string>Manual</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Fixed</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QSpinBox" name="elevationPointsSizeSpinBox">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="minimum">
              <number>1</number>
             </property>
             <property name="maximum">
              <number>10</number>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,0,1">
        <item>
         <widget class="QLabel" name="elevationPointsTransparencyLabel">
          <property name="text">
           <string>Transparency:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="elevationPointsTransparencySpinBox"/>
        </item>
        <item>
         <widget class="QLabel" name="elevationPointsTransparencyPercentLabel">
          <property name="text">
           <string>%</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="waterElevationPointsGroupBox">
     <property name="title">
      <string>Water Elevation Points</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QGroupBox" name="waterElevationPointSizeGroupBox">
        <property name="title">
         <string>Size</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QRadioButton" name="waterElevationPointsAutoSizeRadioButton">
           <property name="text">
            <string>Auto</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="waterElevationPointsManualSizeRadioButton">
           <property name="text">
            <string>Manual</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <spacer name="horizontalSpacer_3">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Fixed</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QSpinBox" name="waterElevationPointsSizeSpinBox">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="minimum">
              <number>1</number>
             </property>
             <property name="maximum">
              <number>10</number>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_4">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="0,0,1">
        <item>
         <widget class="QLabel" name="waterElevationPointsTransparencyLabel">
          <property name="text">
           <string>Transparency:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="waterElevationPointsTransparencySpinBox"/>
        </item>
        <item>
         <widget class="QLabel" name="waterElevationPointsTransparencyPercentLabel">
          <property name="text">
           <string>%</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>4</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>GraphicsSettingDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>215</x>
     <y>357</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>GraphicsSettingDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>215</x>
     <y>363</y>
    </hint>
    <hint type="destinationlabel">
     <x>224</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>elevationPointsManualSizeRadioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>elevationPointsSizeSpinBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>38</x>
     <y>80</y>
    </hint>
    <hint type="destinationlabel">
     <x>66</x>
     <y>108</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>waterElevationPointsManualSizeRadioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>waterElevationPointsSizeSpinBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>41</x>
     <y>247</y>
    </hint>
    <hint type="destinationlabel">
     <x>57</x>
     <y>270</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
